plugins/woocommerce/src/Admin/Features/NewProductManagementExperience.php

```diff
@@ -22,6 +23,24 @@ class NewProductManagementExperience {
 	 * Constructor
 	 */
 	public function __construct() {
+		$new_product_experience_param = 'new-product-experience-disabled';
```

Comment:
Might be nice to move this logic out of the constructor and into a method.  Something like `$this->maybe_show_disabled_notice()`.